lib/ansible/modules/network/aos/aos_asn_pool.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/python
+#
+# (c) 2017 Apstra Inc, <community@apstra.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: aos_asn_pool
+author: Damien Garros (@dgarros)
+version_added: "2.3"
+short_description: Manage AOS ASN Pool
+description:
+  - Apstra AOS ASN Pool module let you manage your ASN Pool easily. You can create
+    create and delete ASN Pool by Name, ID or by using a JSON File. This module
+    is idempotent and support the I(check) mode. It's using the AOS REST API
+requirements:
+  - "aos-pyez >= 0.6.0"
+options:
+  session:
+    description:
+      - An existing AOS session as obtained by aos_login module
```

Comment:
Missing full stop.

One all the modules are updates you could raise a single PR to update any reference to another module with the `M(aos_login)`  which will insert a link to the online docs for that module.

e.g.